### DISCUSSION TITLE
How can i specify output folders for responsive images like we did with fonts?

### DISCUSSION THREAD

**Discussion by mybardaklar** [webdiscus/pug-plugin#49] [Q&A]
Hello again,

Actually i described my question very well. Im waiting for you answer.

I ask this because responsive-loader doesnt allow me define the name property as a function. It has to be string. So yes.

Thank you.

**Comment by webdiscus** [comment]
Hi @mybardaklar,

Pug plugin receive a filename generated by `responsive-loader`.
Create please new issue `[FEATURE REQUEST] filename option as function` by [responsive-loader](https://github.com/dazuaz/responsive-loader/issues).

**Reply by mybardaklar** [reply]
Ah, okay then. I created an issue. Thank you so much!
